packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -83,12 +76,13 @@ const chatMessageStyles: ComponentSlotStylesInput<
     right: v.actionMenuPositionRight,
     top: v.actionMenuPositionTop,
     overflow: p.focused ? 'visible' : 'hidden',
-
-    // hide and squash actions menu to prevent accidental hovers over its invisible area
-    opacity: p.focused ? 1 : 0,
-    width: p.focused ? 'auto' : 0,
+    opacity: 0,
+    width: 0,
+    ...(v.open && {
```

Comment:
This is the only source of truth about whether the action menu should be opened.